docs/guides/unbound.md

```diff
@@ -57,7 +57,9 @@ The first thing you need to do is to install the recursive DNS resolver:
 sudo apt install unbound
 ```
 
-**Important**: Download the current root hints file (the list of primary root servers which are serving the domain "." - the root domain). Update it roughly every six months. Note that this file changes infrequently.
+If you are installing unbound from a package manager, it should install the root.hints automatically with the dependency dns-root-data. The root hints will then be automatically updated by your packaged manager. 
```

Comment:
```suggestion
If you are installing unbound from a package manager, it should install the `root.hints` file automatically with the dependency `dns-root-data`. The root hints will then be automatically updated by your package manager. 
```

Reply:
Also add that the config file needs to have the line uncommented if the file is downloaded.